broctl/broctl.spec

```diff
@@ -12,6 +12,7 @@ Source1:        bro.service
 BuildRequires:  cmake >= 2.6.3
 BuildRequires:  gcc-c++
 BuildRequires:  python2-devel
+BuildRequires:  python-rpm-macros
```

Comment:
This actually isn't needed any more since there's a new `zeekctl` package that supersedes this one.